Omnibus.md

```diff
@@ -3126,4 +3126,303 @@ Zombie Walk (1st Circle)
+  event.
+* In order for the event to be legal the event must abide by the valid rulings
+  of the Arbitration Committee.
+
+As an Event Holder you are expected to:
+
+* Place the safety and welfare of players above all else.
+* Show concern and caution towards sick and injured players.
+* Be impartial, consistent, objective, and courteous when making decisions.
+* Accept responsibility for your actions and decisions, as well as those of
+  your staff.
+* Respect the individuality of other players and event holders.
+* Avoid any situations which may lead to or be construed as a conflict of
+  interest.
+* Always be respectful to other members of the community.
+* Respect the land you are using, follow any site rule imposed by the land
+  owner, and leave the site in better condition than you found it.
+*  Keep up to date with the latest rules of the game and principles of their
+  application.
+* Refrain from any form of personal abuse or harassment towards players or
+  other Event Holders.
+* Respect the rights, dignity, and worth of all people involved in the game
+  regardless of their race, gender, sexual orientation, ability, or cultural
+  background.
+
+Throwing a legal event gives an EH certain privileges and responsibilities
+above and beyond a normal player.
+* To validate your event you must confirm it electronically with the official
+  Event List Administrator within one month of the date of the event. Also
+  within that month, you must also send a summary of reports of rules
+  violations and the conclusion of investigations into them, as well as all
+  consequences levied, to the Arbitration Committee if any reports were filed.
+  EHs and the AC have the responsibility to maintain the privacy of those
+  involved in reports to the extent possible.
+* If you release a Realms Magic Item at your event you must submit the name,
+  physical description, and magical description to the Keeper of the List of
+  Realms Magic Items within one month of that event. The actual prop for the
+  item does not need to be distributed to players at that time, but an EH that
+  does not designate a magic item for an event within one month forever loses
+  the opportunity to release an item for the event (see
+    [Section 7.4: Magic Item Rules](#magic-item-rules) below).
+* As an EH you qualify to attend the Event Holders' Council meeting for the
+  year following your event (see
+  [Section 7.5: The Event-Holders' Council](#the-event-holders-council) below).
+
+## 7.3: Arbitration Committee ## {#arbitration-committee}
+
+The Arbitration Committee is a group of five members and two alternates
+nominated and elected yearly at the EHC that seeks to resolve issues by
+speaking or meeting with the affected individuals and, in the case where
+actions are detrimental to the community, levy penalties against individuals.
+The Arbitration Committee can be reached at arbitration@lists.realmsnet.net.
```

Comment:
`<arbitration@lists.realmsnet.net>` will turn it into a link.  Alternately, `[arbitration@lists.realmsnet.net](arbitration@lists.realmsnet.net)` works.  The first is shorthand for when the link text matches the link address, and I used it for another e-mail address somewhere.